### PR TITLE
Add Xenial Dockerfile

### PR DESCRIPTION
**Pull request by zabio3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 - [`1.7.9.5`, `1.7.9`, `1.7`, `latest` (*debian/Dockerfile*)](https://github.com/jiangwenyuan/docker-nuster/blob/30fdeb1800a885c4ef1c67dfbb5b3906c7d05e10/debian/Dockerfile)
 - [`1.7.9.5-alpine`, `1.7.9-alpine`, `1.7-alpine` (*alpine/Dockerfile*)](https://github.com/jiangwenyuan/docker-nuster/blob/30fdeb1800a885c4ef1c67dfbb5b3906c7d05e10/alpine/Dockerfile)
+- [`16.04`, `xenial` (*xenial/Dockerfile*)](https://github.com/jiangwenyuan/docker-nuster/blob/30fdeb1800a885c4ef1c67dfbb5b3906c7d05e10//xenial/Dockerfile)
```
Comment (jiangwenyuan) [inline README.md]: `16.04` should be `1.7.9.5-xenial`

**Comment by jiangwenyuan** [comment]
Hi, thank you very much, I've added a comment, could you please take a look?

**Comment by zabio3** [comment]
Thanks for nuster created!! Modify README (remove extra slash)

**Comment by jiangwenyuan** [comment]
@zabio3 Hi, could you please change the version number?

```
[`16.04`, `xenial` (*xenial/Dockerfile*)](https://github.com/jiangwenyuan/docker-nuster/blob/30fdeb1800a885c4ef1c67dfbb5b3906c7d05e10//xenial/Dockerfile) 
```
should be

```
[`1.7.9.5-xenial`, `1.7.9-xenial`, `1.7-xenial` (*alpine/Dockerfile*)](https://github.com/jiangwenyuan/docker-nuster/blob/30fdeb1800a885c4ef1c67dfbb5b3906c7d05e10/alpine/Dockerfile) 
```

**Comment by jiangwenyuan** [comment]
Thank you so much:)

**Pull request merged**